### PR TITLE
Deafness renamed to Hearing Loss

### PR DESCRIPTION
**Pull request by matentzn**
See https://github.com/monarch-initiative/mondo/issues/551

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -10800,9 +10800,10 @@ is_obsolete: true
 
 [Term]
 id: MONDO:0000912
-name: autosomal recessive nonsyndromic deafness 5
+name: autosomal recessive nonsyndromic hearing loss 5
 def: "An autosomal recessive nonsyndromic deafness that has material basis in variation in the chromosome region 14q12." [DOID:0110507, PMID:8944017]
 synonym: "autosomal recessive deafness 5" EXACT [DOID:0110507]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo]: I realized that we should change the synonym scope to NARROW for all of the synonyms that have deafness in the name. Would you mind doing this too @matentzn?
thanks!

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -199463,15 +199461,15 @@ is_a: MONDO:0003847 {source="MESH:C564473/inferred"} ! Mendelian disease
...
+name: X-linked hereditary sensory and autonomic neuropathy with hearing loss
 def: "This syndrome is characterized by the association of an axonal sensory and autonomic neuropathy with hearing loss." [Orphanet:139583]
 subset: gard_rare {source="GARD:0012731"}
 subset: ordo_disease {source="Orphanet:139583"}
 synonym: "auditory neuropathy, X-linked, 1, with peripheral sensory neuropathy" RELATED [OMIM:300614]
 synonym: "deafness, X-linked 5" RELATED [GARD:0012731, MONDO:Lexical, OMIM:300614]
 synonym: "DFNX5" RELATED ABBREVIATION [MONDO:Lexical, OMIM:300614]
 synonym: "X-linked auditory neuropathy with peripheral sensory neuropathy type 1" EXACT [Orphanet:139583]
-synonym: "X-linked hereditary sensory and autonomic neuropathy with deafness" EXACT CLINGEN_PREFERRED []
+synonym: "X-linked hereditary sensory and autonomic neuropathy with deafness" NARROW CLINGEN_PREFERRED [OMIM:300614]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo]: there are some duplicates. I manually fixed a few of these. I think you tried to run the command to combine these but it seems that a lot of them were missed for some reason. I can manually fix these if needed.

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -208346,8 +208345,9 @@ property_value: confidence "0.18928571428571428" xsd:double
 
 [Term]
 id: MONDO:0010764
-name: deafness, Y-linked 1
+name: hearing loss, Y-linked 1
 synonym: "deafness, Y-linked 1" EXACT [MONDO:Lexical, OMIM:400043]
+synonym: "deafness, Y-linked 1" NARROW []
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo]: the old labels/now synonyms should have the OMIM ID as the dbxref. I started manually fixing these, I'm not sure if it is possible to automate this?
If not, I'll fix them manually.

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
a few fixes are needed, I can help do some of these manually, if needed.
Thank you!

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (matentzn) [reply src/ontology/mondo-edit.obo]: That is a bit separate but yeah, I can do that.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: THANK YOU! If it is too much to add this to your plate, I can do it manually. Let me know.

Comment (matentzn) [reply src/ontology/mondo-edit.obo:10805]: Ok, for the classes in this branch, under MONDO:0019497, I now made sure that _all_ previously exact synonyms are now NARROW. Double-check though I did it right.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (matentzn) [reply src/ontology/mondo-edit.obo:199472]: I dont think there are duplicates - some synonyms were simply exact and narrow at the same time? Or did youmean something else?

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (matentzn) [reply src/ontology/mondo-edit.obo:208350]: Yeah, that was fun. I think I did it.

**Review by nicolevasilevsky — APPROVED** [review]

**Comment by nicolevasilevsky** [comment]
thanks for your help with this @matentzn! I did all the remaining fixes manually.

**Pull request merged**